CryptoPrices/Sources/MyCoin/Sources/MyCoin/CoinStatisticsSection/CoinStatisticsSection.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  CoinStatisticsSection.swift
+//  MyCoin
+//
+//  Created by Khanh on 03/01/2023.
+//
+
+import Styleguide
+import SwiftUI
+
+struct CoinStatisticsSection: View {
+
+    var body: some View {
+        VStack {
+            setUpCoinStatisticsItemView(
+                title: Strings.MyCoin.MarketCap.title,
+                price: coinDetailItem.marketCap,
+                isPriceUp: coinDetailItem.isMarketCapChangePercentage24HUp,
+                percentage: coinDetailItem.marketCapChangePercentage24H
+            )
+            setUpCoinStatisticsItemView(
+                title: Strings.MyCoin.Ath.title,
+                price: coinDetailItem.ath,
+                isPriceUp: coinDetailItem.isAthChangePercentageUp,
+                percentage: coinDetailItem.athChangePercentage
+            )
+            setUpCoinStatisticsItemView(
+                title: Strings.MyCoin.Atl.title,
+                price: coinDetailItem.atl,
+                isPriceUp: coinDetailItem.isAtlChangePercentageUp,
+                percentage: coinDetailItem.atlChangePercentage
+            )
+        }
+        .padding(16.0)
+    }
+
+    private let coinDetailItem: CoinDetailItem
+
+    init(coinDetailItem: CoinDetailItem) {
+        self.coinDetailItem = coinDetailItem
+    }
+}
+
+private extension CoinStatisticsSection {
+
+    func setUpCoinStatisticsItemView(
+        title: String,
+        price: Decimal,
+        isPriceUp: Bool,
+        percentage: Double
+    ) -> some View {
+        HStack {
+            VStack(alignment: .leading) {
+                Text(title)
+                    .foregroundColor(Colors.textMedium.swiftUIColor)
+                    .font(Fonts.Inter.medium.textStyle(.callout))
+                    .frame(height: 22.0)
+                
+                Text(price, format: .dollarCurrency)
+                    .foregroundColor(Colors.titleMedium.swiftUIColor)
+                    .font(Fonts.Inter.medium.textStyle(.callout))
+                    .frame(height: 28.0)
+            }
```

Comment:
Please help me confirm the font of the `title` here. It seems that it should have one that differs from the one of `price`, which however these show both use the same `Fonts.Inter.medium.textStyle(.callout)`

Also check again the necessity of setting the frame for these texts (`.frame(height: 22.0)` and `.frame(height: 28.0)`) 🙏

Reply:
Good catch 🙌 . The title should have the `caption` font (size 12.0). I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/84/commits/83786ad9d575b07d0f8de4b6c7c0030af3b78667.

>Also check again the necessity of setting the frame for these texts (.frame(height: 22.0) and .frame(height: 28.0)) 🙏

The frame for text is needed here :D. Because in Figma, I don't see the accuracy px space between the title text and price text, as in the following image below:
<img width="383" alt="Screenshot 2023-01-04 at 15 18 59" src="https://user-images.githubusercontent.com/25881847/210513608-2c9574d0-7eca-45ca-b5a3-36e5cfd9d0a8.png">
I set their height frame so they can have their natural padding like the design.

Here is without setting their frame:

<img width="383" src="https://user-images.githubusercontent.com/25881847/210514178-13c328f4-3e73-43e4-8959-03e42efd3a8c.png">

Please correct me if I was wrong or misunderstood your point 🙏 .

Reply:
I think in this case, we can make it simple by adding small vertical padding or providing a custom `spacing` for the `VStack` that contains those texts.
This is the screenshot when I run with `.padding(.vertical, 4.0)` for both texts:
![Simulator Screen Shot - iPhone 14 - 2023-01-04 at 16 13 32](https://user-images.githubusercontent.com/17972674/210522215-d4fedc48-dcbb-402c-8fca-0dcf22a9b695.png)

It looks almost the same as the POW.

It's up to you to choose which approach, but to me adding padding (or setting `VStack`'s spacing) instead of fixing the frame is less font-dependent 🙏